### PR TITLE
Updated Index.htm

### PR DESCRIPTION
**Pull request by YussaMoney**
I updated the index file, re-write and do some changings to the styling through css.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -28,25 +28,24 @@ <h3><button>Sign up</button></h3>
 
     <main>
         <section class="box1"></section>
-        
-        <section class="box3"></section>
-        <p>this is some subtext under an illustration or image</p>
         <div class="box2"><img src="./image/IMG-20220320-WA0110.jpg" alt="an animation" width="125px;" height="117px;"></div>
```
Comment (teekaytech) [inline index.html]: You have some inline CSS here. Always ensure that all CSS declarations are made in CSS files. Kindly update this tag.

File: index.html [hunk 2]
```diff
@@ -28,25 +28,24 @@ <h3><button>Sign up</button></h3>
...
-            This is an inspiring qoute, or a testimonial from a customer. Maybe 
+            <p class="quote">This is an inspiring qoute, or a testimonial from a customer. Maybe 
             it's just filling up space, or maybe people will actually read it.
-            Who knows? All I know is that it looks nice.
-            <h4>-Thor, God of Thunder</h4>
+            Who knows? All I know is that it looks nice.</p>
+            <p class="author">-Thor, God of Thunder</p>
+            <h4>Call to action! It's time!</h4>
+            <p class="content">Sign up for our product by clicking that button right over there!</p>
+            <h3><button>Sign up</button</h3>
```
Comment (teekaytech) [inline index.html]: Why did you wrap a button inside the header (h3) tag? Semantically, [the `<header>` element represents a container for introductory content or a set of navigational links](https://www.w3schools.com/tags/tag_header.asp). Therefore, they should not be used where they are not needed, and I think it's not needed in this regard.

File: index.html (hunk 1 above)
Comment (teekaytech) [inline index.html]: Always name your images such that their names are meaningful and related to the context in which they are used. Kindly change the name of your images.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello Yusuff :wave:

__This is a brilliant effort from you. You've really done an amazing job so far. In this PR, here are my major highlights__

- To a reasonable extent, you've used semantic html tags :+1: 

__However, you need to make some required changes before moving on to the next assignment.__
> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it :blush: 

### **Required Changes** :recycle: 
Check the comments under the review.

- I dropped some comments above. Kindly make changes as indicated.
- The `README.md` file is missing in your project. Kindly add a readme file. You may use [this](https://github.com/microverseinc/readme-template/blob/master/README.md) template to make things easier.

* * *
**Please, do not open a new Pull Request for re-reviews. You should use this same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@teekaytech) so I can be notified about your comments and respond._

_**Please make the required changes and call my attention when you are done.**_

* * *

_Reviewed by:_ [**Taofeek Olalere**](https://github.com/teekaytech)
__Cheers & Happy coding :smiley:__

File: index.html (hunk 1 above)
Comment (YussaMoney) [reply index.html]: I have it there to style the image directly rather than going into CSS files but nevertheless changes have been made.

Comment (YussaMoney) [reply index.html]: Done.

File: index.html (hunk 2 above)
Comment (YussaMoney) [reply index.html]: changes have been made

**Review by teekaytech — APPROVED** [review]
# Status: Approved :green_circle: 

### Congratulations :tada: :clinking_glasses: 

**This project has passed all the mandatory requirements :clap: 
It is therefore approved :heavy_check_mark:**

**Comment(s)**
- In your next project, you should let your pull request title and message be more descriptive.
- Also, let your commit messages be more descriptive. [Check this article](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8) to know more.
- Lastly, always add a live demo link to your project readme. Take note of this against subsequent projects.

Again, congratulations, you can merge to the default branch.

**_Good luck with your next task._**

* * * 
**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Pull request merged**